### PR TITLE
[VL][DOC] Refine the document about parquet write

### PR DESCRIPTION
**Pull request by JkSelf**
## What changes were proposed in this pull request?

Refine the document about parquet write.

## How was this patch tested?

No need.

### CODE REVIEW

**Comment by JkSelf** [comment]
@zhouyuan Please help to review. Thanks.

File: docs/velox-backend-limitations.md [hunk 1]
```diff
@@ -127,7 +138,7 @@ Velox does not create table as select, e.g.,
 ```scala
 spark.range(100).toDF("id")
   .write
-  .format("velox")
+  .format("parquet")
   .saveAsTable("velox_ctas")
 ```
```
Comment (zhouyuan) [inline docs/velox-backend-limitations.md:144]: can you please also add one example on the working case(static partition write)?

File: docs/velox-backend-limitations.md [hunk 2]
```diff
@@ -86,11 +86,22 @@ Exception occurs when Velox TableScan is used to read files with unsupported com
 | DWRF        | Y    | Y    | Y    | Y      | Y   | Y   | N    |
 
 
-### Parquet Write
+### Native Write
 
-#### Offload hive file format to velox (offload)
+#### Offload native write to velox (offload)
```
Comment (zhouyuan) [inline docs/velox-backend-limitations.md:91]: remove` (offload)` in the end ?

File: docs/velox-backend-limitations.md [hunk 3]
```diff
@@ -86,11 +86,22 @@ Exception occurs when Velox TableScan is used to read files with unsupported com
...
 
-We implemented the insert into command by overriding HiveFileFormat in Vanilla spark. And you need to ensure preferentially load the Gluten jar to overwrite the jar of vanilla spark. Refer to [How to prioritize loading Gluten jars in Spark](https://github.com/oap-project/gluten/blob/main/docs/developers/NewToGluten.md). It should be noted that if the user also modifies the HiveFileFormat, the user's changes may be overwritten.
+We have implemented support for Parquet and Hive file formats, as well as static partition writing. This functionality was achieved by overriding the following vanilla Spark classes. And you need to ensure preferentially load the Gluten jar to overwrite the jar of vanilla spark. Refer to [How to prioritize loading Gluten jars in Spark](https://github.com/oap-project/gluten/blob/main/docs/velox-backend-troubleshooting.md#incompatible-class-error-when-using-native-writer). It should be noted that if the user also modifies the overriding classes, the user's changes may be overwritten.
```
Comment (zhouyuan) [inline docs/velox-backend-limitations.md:93]: "implemented write support" ?

**Review by zhouyuan — COMMENTED** [review]
thanks for the improvement!

**Comment by JkSelf** [comment]
@zhouyuan Updated. Please help to review again. Thanks.

**Review by zhouyuan — APPROVED** [review]
👍

**Pull request merged**